webrtc/RTCCertificate-postMessage.html

```diff
@@ -73,5 +73,28 @@
         assert_throws("InvalidAccessError", () => { new RTCPeerConnection({certificates: [certificate2]}) });
         iframe.remove();
     }, "Check cross-origin created RTCCertificate");
+
+    promise_test(async t => {
+        let certificate1 = await  RTCPeerConnection.generateCertificate({ name: 'ECDSA', namedCurve: 'P-256' });
+        let certificate2 = await  RTCPeerConnection.generateCertificate({ name: 'ECDSA', namedCurve: 'P-256' });
+
+        let url = "resources/RTCCertificate-postMessage-iframe.html";
+        let iframe = await with_iframe(url);
+
+        let promise = new Promise((resolve, reject) => {
+            window.onmessage = (event) => {
+                resolve(event.data);
+            };
+            t.step_timeout(() => reject("Timed out waiting for frame to send back certificate"), 5000);
+        });
+        iframe.contentWindow.postMessage(certificate1, "*");
+        let certificate3 = await promise;
+        iframe.remove();
+
+        const pc = new RTCPeerConnection({ certificates: [certificate1, certificate2] });
+        pc.setConfiguration({ certificates: [certificate1, certificate2] });
+        assert_throws("InvalidModificationError", () => { pc.setConfiguration({ certificates: [certificate3, certificate2] }) });
+        assert_throws("InvalidModificationError", () => { pc.setConfiguration({ certificates: [certificate2, certificate1] }) });
```

Comment:
This one throws because sequence is different, right? That's also tested in the other file, and shouldn't be retested here.